contrib/localnet/docker-compose-import-upgrade.yml

```diff
@@ -0,0 +1,55 @@
+version: "3"
+
+# This docker-compose redefine the services:
+# - ZetaChain with 2 nodes (zetacore0, zetacore1) using the upgrade option for cosmovisor
+# - ZetaChain observer set with 2 clients (zetaclient0, zetaclient1) using the background option
+# - Orchestrator with upgrade option
+
+services:
+  zetacore0:
+    image: zetanode:old
```

Comment:
Please add a comment: "note that `zetanode:old` tag is built in `make zetanode-upgrade`"